apps/crl.c

```diff
@@ -161,6 +162,9 @@ int crl_main(int argc, char **argv)
         case OPT_VERIFY:
             do_ver = 1;
             break;
+        case OPT_DATE_FORMAT:
+            date_format = 1;
```

Comment:
You need to decide what you want to do here...
You have specifies this as 's' above? If it is just a '-' then you need to update the string also to indicate what it does.. Currently it changes it to the new format..

Reply:
it looks like you want to use set_date_format() here?